.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't understand with this PR even mentions changes to files located in .idea sub-directory
since you added it the .gitignore file !?

Reply:
Let it be for this time